docs/de-DE/config/README.md

```diff
@@ -845,6 +846,33 @@ The `rust` module shows the currently installed version of Rust. Das Modul wird
 symbol = "⚙️ "
 ```
 
+## Terraform
+
+Das `terraform`-Modul zeigt den aktuellen Terraform-Workspace und Version.
+By default the terraform version is not shown, since this is slow on current versions of terraform when a lot of plugins are in use.
+Das Modul wird nur dann angezeigt, wenn eine der folgenden Bedingungen zutrifft:
+
+- Das aktuelle Verzeichnis enthält eine `.terraform`-Verzeichnis
+- Das aktuelle Verzeichnis enthält eine Datei mit der `.tf`-Erweiterung
+
+### Optionen
+
+| Variable       | Standardwert  | Beschreibung                                               |
+| -------------- | ------------- | ---------------------------------------------------------- |
+| `symbol`       | `"💠 "`       | Symbol das vor den Terraform-Workspace angezeigt wird.     |
+| `style`        | `"bold 105"`  | Stil für dieses Modul.                                     |
+| `disabled`     | `false`       | Deaktiviert das `terraform`-Modul.                         |
+| `show_version` | `false`       | Ziegt den Terraform Version. Langzam in grossen Workspace. |
+
+### Beispiel
+
+```toml
+# ~/.config/starship.toml
+
+[terraform]
+symbol = "🏎💨 "
+```
+
```

Comment:
I don't think Crowdin likes pre-translated docs.
I assume that Crowdin will either overwrite this with the english translation, or still mark it untranslated.
Same with the other docs, it probably expects them to be untouched.

Reply:
I can easily revert back to the original text if that's preferred.

Reply:
I pushed new commits which remove the translations so they can be managed through crowdin